### PR TITLE
Add more operator support and tests

### PR DESCRIPTION
**Pull request by lly-zero-one**
Add more operator tests for cuda_codegen
Also support the relu

### CODE REVIEW

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 1]
```diff
@@ -392,6 +392,12 @@ Tensor TensorExprKernel::ComputeValue(torch::jit::Value* v) {
           "aten_round", v, [](const Expr& a) { return round(a); });
     } break;
 
+    case aten::relu: {
```
Comment (protonu) [inline torch/csrc/jit/tensorexpr/kernel.cpp]: @lly-zero-one I checked in a version of ReLu yesterday. It should be in the repo now:

    case aten::relu: {
      return ComputeOneOperand("aten_relu", v, [](const Expr& a) {
        Expr zero_cond = CompareSelect::make(a, Expr(0.0f), kLT);
        return ifThenElse(zero_cond, Expr(0.0f), a);
      });
    } break;
Though not sure which version is better perf-wise.

Comment (lly-zero-one) [reply torch/csrc/jit/tensorexpr/kernel.cpp]: Thanks, let me pull it out.

File: torch/csrc/jit/passes/tensorexpr_fuser.cpp [hunk 2]
```diff
@@ -72,6 +72,7 @@ bool isSupported(Node* node) {
     case aten::sqrt:
     case aten::rsqrt:
     case aten::abs:
+    case aten::relu:
```
Comment (protonu) [inline torch/csrc/jit/passes/tensorexpr_fuser.cpp]: after a rebase this may not be needed

Comment (lly-zero-one) [reply torch/csrc/jit/passes/tensorexpr_fuser.cpp]: I expect it should support by both now?

**Review by protonu — APPROVED** [review]

**Pull request merged**